extensions/notebook/src/protocol/notebookUriHandler.ts

```diff
@@ -88,9 +95,18 @@ export class NotebookUriHandler implements vscode.UriHandler {
 		}
 	}
 
-	private download(url: string): Promise<string> {
+	private download(url: string, authToken?: string): Promise<string> {
```

Comment:
This never resolves, does it?

Reply:
![image](https://user-images.githubusercontent.com/1465785/83802997-9ed43500-a660-11ea-80fe-30869e48ca0a.png)


am I missing something?

Reply:
Oops, it *was* on 110, now it's on 126

![image](https://user-images.githubusercontent.com/28519865/83804320-ccba7900-a662-11ea-8b7b-dfd77b6b076b.png)

Reply:
Oh whoops, didn't notice this.